nessie/src/main/java/org/apache/iceberg/nessie/UpdateableReference.java

```diff
@@ -60,6 +60,8 @@ public String getHash() {
     return reference.getHash();
   }
 
+  /** @deprecated will be removed in 1.3.0 */
+  @Deprecated
```

Comment:
What's calling this one?

Reply:
I deprecated it instead of removing it because it is a public method. 
Now, I realise that the class is not public. I removed this API now.